### ISSUE TITLE
pprof endpoints

### ISSUE THREAD

**Issue by brancz** [BuoyantIO/emojivoto#103]
Hey :wave: 

We'd love to use emojivoto as an example workload in @PolarSignals. To produce some data for Polar Signals Continuous Profiler we'de love to add pprof endpoints to each of the components. Would you be open to us adding the pprof endpoints in a similar manner as the prometheus endpoint? If you prefer we can also generalize them into a single "telemetry" port.

Let me know what you think, we're happy to take care of it!

**Comment by olix0r** [comment]
@brancz This sounds like a great addition! We'd welcome this change.

+1 to serving these endpoints on a separate admin server if they're not already

**Comment by brancz** [comment]
There is already the [`PROM_PORT`](https://github.com/BuoyantIO/emojivoto/blob/ed0be99b39ac6c9f2631c057dfd3f36057be70b1/emojivoto-emoji-svc/cmd/server.go#L25). I would propose to rename it to `TELEMETRY_PORT` and serve both the prometheus and the pprof endpoints under it, but don't feel strongly about it whatsoever, so if you prefer a separate `PPROF_PORT`, I'm totally ok with that as well.

**Comment by olix0r** [comment]
@brancz Yeah, let's unify it as either `TELEMETRY_PORT` or `ADMIN_PORT`. Thanks!

**Comment by brancz** [comment]
I started working on this, but I think I have mismatching versions of protoc and plugins from what this is built with. Do you mind if I update things to the latest version or let me know which version is being used to build this?

**Comment by alpeb** [comment]
Thanks @brancz, upgrading to the latest protoc version sounds great!
